src/timeLineSettingsModel.ts

```diff
@@ -285,20 +236,73 @@ export class CellsSettingsCard extends Card {
         },
     });
 
+    cellsGeneralGroup = new Group({
+        name: "cellsGeneralGroup",
+        displayName: "General",
+        displayNameKey: "Visual_General",
+        slices: [this.strokeWidth, this.gapWidth, this.enableManualSizing, this.width, this.height],
+    })
+
+    fillSelected = new formattingSettings.ColorPicker({
+        name: "fillSelected",
+        displayName: "Color",
+        displayNameKey: "Visual_Color",
+        value: { value: CellsSettingsCard.FillSelectedDefaultColor },
+    });
+
+    strokeSelected = new formattingSettings.ColorPicker({
+        name: "strokeSelected",
+        displayName: "Stroke color",
+        displayNameKey: "Visual_StrokeColor",
+        value: { value: "#333444" },
+    })
+
+    showEdges = new formattingSettings.ToggleSwitch({
+        name: "showEdges",
+        displayName: "Show edges",
+        displayNameKey: "Visual_ShowEdges",
+        value: true,
+    });
+
+    edgeColor = new formattingSettings.ColorPicker({
+        name: "edgeColor",
+        displayName: "Edge color",
+        displayNameKey: "Visual_EdgeColor",
+        value: { value: "#808080" },
+    });
+
+    selectedCellsGroup = new Group({
+        name: "selectedCellsGroup",
+        displayName: "Selected cells",
+        displayNameKey: "Visual_SelectedCells",
+        slices: [this.fillSelected, this.strokeSelected, this.showEdges, this.edgeColor],
+    });
+
+    fillUnselected = new formattingSettings.ColorPicker({
+        name: "fillUnselected",
+        displayName: "Color",
+        displayNameKey: "Visual_Color",
+        value: { value: CellsSettingsCard.FillUnselectedDefaultColor },
+    });
+
+    strokeUnselected = new formattingSettings.ColorPicker({
+        name: "strokeUnselected",
+        displayName: "Stroke color",
+        displayNameKey: "Visual_StrokeColor",
+        value: { value: "#333444" },
+    });
+
+    unselectedCellsGroup = new Group({
+        name: "unselectedCellsGroup",
+        displayName: "Unselected cells",
+        displayNameKey: "Visual_UnselectedCells",
+        slices: [this.fillUnselected, this.strokeUnselected],
+    });
+
     name: string = "cells";
     displayName: string = "Cells";
     displayNameKey: string = "Visual_Cells";
-    slices = [
-        this.fillSelected,
-        this.strokeSelected,
-        this.fillUnselected,
-        this.strokeUnselected,
-        this.strokeWidth,
-        this.gapWidth,
-        this.enableManualSizing,
-        this.width,
-        this.height,
-    ];
+    groups = [this.cellsGeneralGroup, this.selectedCellsGroup, this.unselectedCellsGroup];
```

Comment:
better to keep naming pattern going from one object to another: cells -> selected -> group = cellsSelectedGroup, not selectedCellsGroup

Reply:
Renamed.